### ISSUE TITLE
FAILS TO LOAD in strict-mode. How to fix, included in description

### ISSUE THREAD

**Issue by NawarA** [NaturalNode/natural#474]
The software fails to load when `'use strict'` is enabled. 

```javascript
/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/suffix_rules.js:28
SuffixRules = function() {
            ^

ReferenceError: SuffixRules is not defined
    at Object.<anonymous> (/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/suffix_rules.js:28:13)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/stemmer_id.js:31:19)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
```

```javascript
/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/prefix_rules.js:43
PrefixRules = function() {
            ^

ReferenceError: PrefixRules is not defined
    at Object.<anonymous> (/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/prefix_rules.js:43:13)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/nawar/dev/API/node_modules/natural/lib/natural/stemmers/indonesian/stemmer_id.js:32:19)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
```

These two lines are [not written safe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). These variables are missing declarations, so the software throws.

### How To Fix
In file: `/natural/lib/natural/stemmers/indonesian/prefix_rules.js:43:13`

change this:
```javascript
PrefixRules = function() {

```
to this:
```javascript
function PrefixRules() {
```

AND In file: `/natural/lib/natural/stemmers/indonesian/suffix_rules.js:28:13`

from
```javascript
SuffixRules = function() {

```
to
```javascript
function SuffixRules() {
```

I can confirm that changing these two lines of code makes the software work in strict mode. Please fix this bug

**Comment by Hugo-ter-Doest** [comment]
Fixed it. Thanks for your help!

**Comment by NawarA** [comment]
@Hugo-ter-Doest Thanks! When do you plan to do an `npm release` with the patch?

**Comment by Hugo-ter-Doest** [comment]
Some time around Christmas.

**Comment by NawarA** [comment]
@Hugo-ter-Doest Thank you dude! I'm looking forward to the patch! I have to keep our entire node.js server from using `--strict_mode` b/c of this bug. Enabling `--use_strict` in V8 improves machine code performance.

Looking forward to the patch, so I can re-enable `use_strict` in production. Thank you!!

**Comment by Hugo-ter-Doest** [comment]
I published version 0.6.3 to NPM

**Comment by NawarA** [comment]
Thanks @Hugo-ter-Doest  you're awesome!
